backend/packages/Upgrade/src/init/seed/initMetrics.ts

```diff
@@ -1,13 +1,14 @@
 import Container from 'typedi';
 import { MetricService } from '../../api/services/MetricService';
 import { env } from '../../env';
+import { UpgradeLogger } from '../../lib/logger/UpgradeLogger';
 
 export function InitMetrics(): Promise<any> {
   const metricService: MetricService = Container.get(MetricService);
   // Init default metrics in system
   if (env.initialization.metrics) {
     try {
-      return metricService.saveAllMetrics(JSON.parse(env.initialization.metrics));
+      return metricService.saveAllMetrics(JSON.parse(env.initialization.metrics), new UpgradeLogger());
```

Comment:
Are you going to update this in another PR?

Reply:
yes